src/lightning_app/components/serve/python_server.py

```diff
@@ -105,7 +106,7 @@ def predict(self, request):
         self._input_type = input_type
         self._output_type = output_type
 
-    def setup(self) -> None:
+    def setup(self, *args, **kwargs) -> None:
```

Comment:
why is this needed?

Reply:
in case users wants to use the args from the run method.

Reply:
to give the context, this is a function users would override - like an abstract method.